check_powerdns_rec.py

```diff
@@ -19,17 +19,18 @@
 ################################################################################
 
 __author__ = 'Remi Frenay <rf@worldstream.nl>'
-__version__ = '1.0.0'
+__version__ = '1.1.0'
 __plugin_name__ = 'check_powerdns_rec.py'
 
 import os
 import pickle
 import re
 import subprocess
+import requests
+import json
 import sys
 import time
 
-# Clean area interfaces
 pdns_tool = 'rec_control'
```

Comment:
Could be part of `PowerDnsCtrlTool` class. See remark below.

Reply:
Done